transport/http/outbound.go

```diff
@@ -288,18 +288,22 @@ func (o *Outbound) callWithPeer(
 
 	span.SetTag("http.status_code", response.StatusCode)
 
+	bothResponseError := fromAcceptValue(response.Header.Get(BothResponseErrorHeader))
```

Comment:
is it possible to put this directly before the if statement that uses it?  We could also inline it but that might reduce readability.

Reply:
Done